api/scpca_portal/readme_file.py

```diff
@@ -30,6 +34,6 @@ def get_file_contents(download_config: Dict, project) -> str:
             "date": utils.get_today_string(),
             "download_config": download_config,
             "contents_template": contents_template,
-            "projects": [project],
+            "projects": projects if isinstance(projects, QuerySet) else [projects],
```

Comment:
```suggestion
            "projects": projects,
```

You'll need to update other callers of this method so that it matches the iterable type.

Reply:
See at 68053c986dc5b971a57bbee8039042a741dc20c8.